org.civicrm.reqangular/js/src/common/services/api/contract.js

```diff
@@ -43,6 +43,15 @@ define([
 
             return contracts;
           });
+      },
+      getContactsWithContractsInPeriod: function (startDate, endDate) {
```

Comment:
you're not giving any documentation to this method

Reply:
Done.